File: lib/utils/animations/entrance/throwback.dart
import '../base/animated_text_state.dart';

AnimatedTextState evaluateThrowback(double t) {
  final scale = 4.0 - 3.0 * t;
  final opacity = t < 0.3 ? t / 0.3 : 1.0;
  return AnimatedTextState(scale: scale, opacity: opacity);
}

File: android/app/src/main/kotlin/com/typography/animations/entrance/Throwback.kt
package com.typography.animations.entrance

import com.typography.AnimatedTextState

fun evaluateThrowback(t: Float): AnimatedTextState {
    val s = 4.0f - 3.0f * t
    val o = if (t < 0.3f) t / 0.3f else 1.0f
    return AnimatedTextState(scale = s, opacity = o)
}
